test/onnx/test_pytorch_onnx_onnxruntime.py

```diff
@@ -2726,6 +2726,36 @@ def forward(self, input, indices):
         indices = torch.tensor([[1, 0], [0, 1], [0, 1]], dtype=torch.int64)
         self.run_test(GatherModel(), input=(input, indices))
 
+    @skipIfUnsupportedMinOpsetVersion(11)
+    def test_gather_constant_fold(self):
```

Comment:
Move these tests to utility tests?

Reply:
Maybe also describe each test is testing which case (rank 0, etc.) in comments.

Reply:
utility tests only test if the node has been constant-folded, not if it has been done in the correct way. Here the issue is that constant_folding was outputting the incorrect result.

Reply:
Actually they could test both. Anyways, constant folding is a utility.